--- xs-config.make.orig	2010-07-20 15:01:42.000000000 -0500
+++ xs-config.make	2010-07-27 20:14:57.000000000 -0500
@@ -5,14 +5,13 @@
 ## how this works...
 ##
 earlyset: rsyslog.conf motd sysctl.conf ssh/sshd_config \
-     sysconfig/named sysconfig/init \
-     sysconfig/iptables-config sysconfig/squid \
-     sudoers rssh.conf php.ini sysconfig/httpd \
-     init.d/squid sysconfig/ejabberd \
-     sysconfig/network-scripts/ifcfg-eth0 sysconfig/network-scripts/ifcfg-eth1 \
-     httpd/conf.d/proxy_ajp.conf httpd/conf.d/ssl.conf
+     sysconfig/init sudoers rssh.conf php.ini sysconfig/httpd \
+     sysconfig/ejabberd httpd/conf.d/proxy_ajp.conf httpd/conf.d/ssl.conf
+
 
-networkset: hosts sysconfig/dhcpd
+networkset: hosts sysconfig/dhcpd sysconfig/named sysconfig/squid \
+     sysconfig/network-scripts/ifcfg-eth0 sysconfig/network-scripts/ifcfg-eth1 \
+     init.d/squid sysconfig/iptables-config 
 
 # Any file that has a ".in"
 # 'template' can be made with this catch-all
